lib/protocol/schema/definitions.capnp

```diff
@@ -37,7 +37,8 @@ struct Schema {
 }
 
 struct DealInfo {
-    id @0 :UInt64;
-    selectorPath @1 :Text;
-    cid @2 :Text;
+    cid @0 :Text;
+    size @1 :UInt32;
+    created @2 :Text; 
+    isPermament @3 :Bool;
```

Comment:
nit: "permanent" might be misleading... it's only stored until the deal expires. maybe "archived" would be better terminology? in any case, we can deal with this later... guessing we'll have to wipe the DB at some point when we move to postgres

Reply:
nice! `archived` is better, I wasn't happy very with permanent but could not find a word